extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -189,6 +189,19 @@ export class AmpAdUIHandler {
       resizeInfo.success = false;
       return Promise.resolve(resizeInfo);
     }
+
+    // Special case: for sticky ads, we enforce 20% size limit and 50% height limit
+    if (this.element_.hasAttribute('sticky')) {
+      const viewport = this.baseInstance_.getViewport();
+      if (
+        newHeight * newWidth >
+          0.2 * viewport.getHeight() * viewport.getWidth() ||
```

Comment:
nit: can we name these numbers something like MAX_SIZE_LIMIT & MAX_HEIGHT_LIMIT

Reply:
done